src/automator/storage.task.spec.js

```diff
@@ -21,29 +21,29 @@ describe('storage task handler', function() {
     scope = $rootScope;
     backend = $httpBackend;
     baseUrl = function(path) {
-      return clbEnv.get('api.document.v0') + '/' + (path ? path : '');
+      return clbEnv.get('api.document.v1', 'https://services.humanbrainproject.eu/storage/v1/api') + '/' + (path ? path + '/' : '');
```

Comment:
I would not provide a default value here, better to fail if the configuration is not up-to-date, IMHO